styles/game.css

```diff
@@ -0,0 +1,17 @@
+#grid-container {
+    padding-bottom: 10px;
+}
+.grid {
+    background-color: #C5D6C6;
+    border-spacing: 0;
+}
+
+.gridCell {
```

Comment:
This is related to comments in your index.html file. Currently elements with the class grid and gridCell will not be created. In your game.js script I believe you want to change your createElement calls all to createElement("div") and set the class equal to grid and gridCell.